core/src/services/s3/core.rs

```diff
@@ -765,6 +765,22 @@ pub struct CompleteMultipartUploadRequest {
     pub part: Vec<CompleteMultipartUploadRequestPart>,
 }
 
+/// Result of MultipartUploadRequest.
+///
+/// Result part on success are ignored.
+#[derive(Default, Debug, Deserialize, Eq, PartialEq)]
+#[serde(default, rename_all = "PascalCase")]
+pub struct CompleteMultipartUploadResult {
+    pub error: Vec<CompleteMultipartUploadResultError>,
+}
```

Comment:
I only define the error parts to minimize the change. The body is discarded before so I assume its content doesn't matter. We can add a field in the future when we need it.